packages/kbn-discover-utils/src/data_types/logs/components/log_level_badge.tsx

```diff
@@ -40,7 +40,7 @@ export const LogLevelBadge = ({
         data-test-subj={`${dataTestSubj}-unknown`}
         css={badgeCss}
       >
-        {logLevel}
+        {logLevel as React.ReactNode}
```

Comment:
logLevel is `{}` for some reason. Might be worth fixing on a level higher, but I didn't want to look into what would fail :)